app/(pages)/profile/page.tsx

```
@@ -1505,6 +1508,23 @@ export default function ProfilePage() {
     stripePayoutsEnabled
   )
 
+  useEffect(() => {
```

ESLint

[error] React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)